protobuf/v1/schema.proto

```diff
@@ -7,10 +7,13 @@ option go_package = "schema/service/v1";
 import "google/protobuf/struct.proto";
 import "google/api/annotations.proto";
 
+message HTTPErrorResponse {
```

Comment:
Is this error code used?

Reply:
This message is used in the http service within a runtime.WithErrorHandler option, which handles non 0 codes from the grpc gateway (otherwise the gateway will serve default gRPC error attributes (code, message, detail messages))
I have kept it within this proto file so it can be used to generate the client side structures in other languages (rather than defining it within the http_handler code)